digital-twins/jsonld-contexts/digital_twins_compound.jsonld

```diff
@@ -0,0 +1,6 @@
+{
+  "@context": [
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/observed_attributes.jsonld",
```

Comment:
name of the context is no longer accurate. maybe call it `usecase.jsonld`, it is globally what it is about.

Reply:
agree